src/android/run.ts

```diff
@@ -10,19 +10,24 @@ import { installApkToDevice, selectDeviceByTarget, selectHardwareDevice, selectV
 import { SDK, getSDK } from './utils/sdk';
 
 export async function run(args: string[]) {
+
   const sdk = await getSDK();
   const apkPath = getOptionValue(args, '--app');
-  const forwardedPorts = getOptionValue(args, '--forward');
-  let ports: Ports | undefined;
 
-  if (forwardedPorts) {
-    const [ device, host ] = forwardedPorts.split(':');
+  const forwardedPorts = getOptionsValue(args, '--forward');
+
+  const ports: Ports[] = [];
+
+  if (forwardedPorts && forwardedPorts.length > 0) {
+    forwardedPorts.forEach((port: string) => {
+    const [ device, host ] = port.split(':');
 
     if (!device || !host) {
       throw new CLIException('Invalid --forward value: expecting <device port:host port>, e.g. 8080:8080');
```

Comment:
Now that `--forward` can take multiple values, it may be useful to print which values were the bad ones.